frontend/src/components/MeetingRoom/ParticipantListItem/ParticipantListItem.tsx

```diff
@@ -1,16 +1,20 @@
-import { Avatar, ListItem, Typography } from '@mui/material';
 import { ReactElement } from 'react';
+import { Avatar, Badge, ListItem, Typography } from '@mui/material';
+import PushPinIcon from '@mui/icons-material/PushPin';
 import { Stream } from '@vonage/client-sdk-video';
 import AudioIndicator from '../AudioIndicator';
+import ParticipantListItemMenu from '../ParticipantListItemMenu';
+import { SubscriberWrapper } from '../../../types/session';
 
 export type ParticipantListItemProps = {
   stream?: Stream;
   initials: string;
   hasAudio?: boolean;
   audioLevel?: number;
-  name?: string;
+  name: string;
```

Comment:
is there any particular reasoning behind enforcing the `name` property at this point?

Reply:
The name props was unnecessarily optional since the empty case is handled in the parent components, probably a remnant from an earlier usage. Adding this as required means we don't have to check it's existence in the code below.